Extensions/PrimitiveDrawing/ShapePainterObject.h

```diff
@@ -81,6 +81,9 @@ class GD_EXTENSION_API ShapePainterObjectBase {
   inline void SetCoordinatesRelative() { absoluteCoordinates = false; }
   inline bool AreCoordinatesAbsolute() { return absoluteCoordinates; }
 
+  inline void SetClearAtEachFrame(bool value = true) { clearAtEachFrame = value; }
```

Comment:
Let's remove the default parameter here, it's not actually used because we're always passing a value (true or false). The default value for existing project must be set at the step of unserialization.